pkg/collector/python/datadog_agent.go

```diff
@@ -287,14 +294,109 @@ var defaultSQLPlanNormalizeSettings = traceconfig.JSONObfuscationConfig{
 		"using_join_buffer",
 		"using_temporary_table",
 		// postgres
+		"Actual Loops",
+		"Actual Rows",
+		"Actual Startup Time",
+		"Actual Total Time",
 		"Alias",
+		"Async Capable",
+		"Average Sort Space Used",
+		"Cache Evictions",
+		"Cache Hits",
+		"Cache Misses",
+		"Cache Overflows",
+		"Command",
+		"Conflicting Tuples",
+		"Constraint Name",
+		"CTE Name",
+		"Custom Plan Provider",
+		"Deforming",
+		"Emission",
+		"Exact Heap Blocks",
+		"Execution Time",
+		"Expressions",
+		"Foreign Delete",
+		"Foreign Insert",
+		"Foreign Update",
+		"Function Call",
+		"Function Name",
+		"Generation",
+		"Group Count",
+		"Grouping Sets",
+		"Group Key",
+		"HashAgg Batches",
+		"Hash Buckets",
+		"Hash Key",
+		"Hash Keys",
```

Comment:
Do you have an example of these?

Reply:
Discussed offline, but a note for future readers. We were unable to discover what values these two fields provide, so we've removed them for this iteration.